scss/_utilities_embedded-media.scss

```diff
@@ -4,6 +4,7 @@
 @mixin vf-u-embedded-media {
   .u-embedded-media {
     height: 0;
+    margin-bottom: map-get($sp-after, p) - map-get($nudges, nudge--p);
```

Comment:
@lyubomir-popov Does that sound right? Or would simply `$spv-outer--medium` work here?

Reply:
@bartaz best to use `$spv-outer--scaleable`, it is used on most other large blocky components like p-card etc.

Reply:
thanks both - I've updated